unstructured/ingest/connector/sql/connector.py

```diff
@@ -0,0 +1,216 @@
+import json
+import typing as t
+import uuid
+from dataclasses import dataclass, field
+
+from unstructured.ingest.enhanced_dataclass import enhanced_field
+from unstructured.ingest.interfaces import (
+    AccessConfig,
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.utils import requires_dependencies
+
+from .schema import (
+    ELEMENTS_TABLE_NAME,
+    DatabaseSchema,
+)
+
+
+@dataclass
+class SqlAccessConfig(AccessConfig):
+    password: t.Optional[str] = enhanced_field(sensitive=True)
```

Comment:
I would add username in here as well, but not mark is as sensitive as it's still a value needed for accessing the db.